gst/nnstreamer/nnstreamer_subplugin.c

```diff
@@ -68,6 +68,7 @@ static subpluginSearchLogic searchAlgorithm[] = {
   [NNS_SUBPLUGIN_DECODER] = NNS_SEARCH_FILENAME,
   [NNS_EASY_CUSTOM_FILTER] = NNS_SEARCH_FILENAME,
   [NNS_SUBPLUGIN_CONVERTER] = NNS_SEARCH_GETALL,
+  [NNS_IF_CUSTOM] = NNS_SEARCH_FILENAME,
```

Comment:
NNS_SEARCH_NO_OP? unless you intend to enable "nnstreamer_customif_whatever.so" to be loaded.

Reply:
Thanks. Changed to `NNS_SEARCH_NO_OP`